test/framework/bootstrap/kind_provider.go

```diff
@@ -121,6 +121,9 @@ func withDockerSockConfig() *kindv1.Cluster {
 			APIVersion: "kind.x-k8s.io/v1alpha4",
 			Kind:       "Cluster",
 		},
+		Networking: kindv1.Networking{
+			IPFamily: kindv1.IPv6Family,
```

Comment:
We should define how to make this info flow down the chain.
Also, are you expecting to support combination where the management cluster in kind has a different  ipFamily than the workload clusters or are we assuming that both are always consistent?

Reply:
> are you expecting to support combination where the management cluster in kind has a different ipFamily than the workload clusters or are we assuming that both are always consistent?

I think in the short term, we should expect them to be consistent. Longer term, when we're really thinking about dual stack, I think you argue that the management cluster IP family should "contain" the workload cluster IP family.